templates/frontend/objects/article_details.tpl

```diff
@@ -134,11 +134,11 @@
 		<div class="entry_details">
 
 			{* Issue cover image *}
-			{if $issue->getLocalizedFileName()}
+			{if $issue->getLocalizedCoverImage()}
 				<div class="item cover_image">
 					<div class="sub_item">
 						<a href="{url page="issue" op="view" path=$issue->getBestIssueId($currentJournal)}">
-							<img src="{$publicFilesDir}/{$issue->getLocalizedFileName()|escape}" alt="{$issue->getLocalizedOriginalFileName()|escape}">
+							<img src="{$publicFilesDir}/{$issue->getLocalizedCoverImage()|escape}">
```

Comment:
shouldn't here the article objects be used and not issue object? then also the getLocalizedFileName would be the right function?